packages/next-ui/Button/LinkOrButton.tsx

```diff
@@ -80,6 +83,12 @@ export const LinkOrButton = React.forwardRef<
             display: { xs: 'inline-flex', [breakpoint]: 'none' },
             alignItems: 'center',
           },
+          typeof disabled !== 'undefined' &&
+            disabled && {
+              opacity: 0.5,
+              filter: 'grayscale(1)',
+              pointerEvents: 'none',
```

Comment:
Can we perhaps use this?

<img width="727" alt="Schermafbeelding 2022-04-07 om 11 41 56" src="https://user-images.githubusercontent.com/1244416/162170761-b67e555b-d251-4253-b174-604aa7b4f85d.png">

Reply:
```
typeof disabled !== 'undefined' &&
            disabled && {},
```

Can also be written like:

```
!!disabled && {}
```

Reply:
Sure!